pennylane/ops/qubit/parametric_ops.py

```diff
@@ -119,6 +119,10 @@ def _controlled(self, wire):
         new_op = CRX(*self.parameters, wires=wire + self.wires)
         return new_op.inv() if self.inverse else new_op
 
+    def simplify(self):
+        theta = self.data[0] % (4 * np.pi)
```

Comment:
Why `(4 * np.pi)`?

Wouldn't it be better to simplify `(2 * np.pi)` to `0`?

Reply:
```
>>> qml.RX.compute_matrix(2*np.pi)
array([[-1.+0.0000000e+00j,  0.-1.2246468e-16j],
       [ 0.-1.2246468e-16j, -1.+0.0000000e+00j]])
```
While this global phase won't show up ordinarily, it will show up if we perform a controlled version of the operation.

Reply:
Ah is this because RX is defined as exp^(i*theta/2)PauliX ?